task/s2i-angular-nginx/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Add git-clone
+add_task git-clone latest
```

Comment:
@jlozano-rh can you please do the following changes?
```suggestion
add_task git-clone 0.7
```

Reply:
@chmouel we would be needing the above as with `git-clone latest`, CI is for sure failing. https://github.com/tektoncd/catalog/pull/1079 got merged just now. This will make the CI green for the tasks which are already there in catalog.